go.mod

```diff
@@ -6,17 +6,17 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo/v2 v2.6.0
 	github.com/onsi/gomega v1.24.1
-	github.com/openshift/api v0.0.0-20221013123531-622889ac07cf
-	github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
+	github.com/openshift/api release-4.11
```

Comment:
Nice :)
IMHO this way (using `release-OCP_Y_VERSION` instead of long version number) seems easier to read or identify the version.

Reply:
well, it's just an intermediate helper., instead of finding out the correct version myself.  `go mod tidy` replaces it with the long version